spec/helpers/ems_cloud_helper/textual_summary_spec.rb

```diff
@@ -57,7 +57,7 @@
   end
 
   describe '#textual_description' do
```

Comment:
What do you think of this?  I'd prefer not to stub the `description` method on https://github.com/ManageIQ/manageiq-ui-classic/pull/5755/files#diff-bb5750a3645104a714926b5306043be6R69
```ruby
  describe '#textual_description' do
    context "EMS instance doesn't support :description" do
      before { instance_variable_set(:@record, FactoryBot.create(:ext_management_system)) }

      include_examples 'textual_description', nil
    end

    context "EMS instance has :description" do
      before { instance_variable_set(:@record, FactoryBot.create(:ems_cloud)) }

      include_examples 'textual_description', :label => _("Description"), :value => "US East (N. Virginia)"
    end
  end
```